### PR TITLE
Create Voting Method Info Page

### LINKED ISSUE #121: 📜 Voting Method Details
Link details links on voting methods selection to their corresponding page (Early voting details, Mail-in voting details, etc.). Currently, the details information is stored as markdown so we'll want to reference Sue's markdown parser to render/style HTML. 

*Currently writing this on my phone lol but message Amit/I for further details

### PR DESCRIPTION
**Pull request by ericcccsliu**
## Status:
:rocket: Ready

## Summary

Closes #121 

## Changes

- Created Voting Method Page
- Populated Voting Method Selection with text from Contentful 
- added padding somewhere i think 
- eeeeeweeeweeek

## Screenshots
Screen Shot 2022-05-01 at 3 04 15 AM
slay
Screen Shot 2022-05-01 at 3 04 35 AM

### CODE REVIEW

File: client/src/pages/PollingPlaceIssue.tsx [hunk 1]
```diff
@@ -28,6 +28,56 @@ const getIssueIndex = (id: string): number => {
...
+  },
+  h1: (props: { children: any }) => {
+    const { children } = props;
+    return (
+      <Heading as="h1" textStyle="h1" fontSize={['28px', '32px']}>
+        {children}
+      </Heading>
+    );
+  },
+  h2: (props: { children: any }) => {
```
Comment (amit-sawhney) [inline client/src/pages/PollingPlaceIssue.tsx:49]: any really should be ReactElement

File: client/src/pages/PollingPlaceIssue.tsx [hunk 2]
```diff
@@ -28,6 +28,56 @@ const getIssueIndex = (id: string): number => {
...
+        borderColor="primary"
+        py={2}
+        pl={2}
+        my={3}
+      >
+        {children}
+      </Heading>
+    );
+  },
+  ul: (props: { children: any }) => {
```
Comment (amit-sawhney) [inline client/src/pages/PollingPlaceIssue.tsx:67]: You may be able to shorthand type this by `props: FC` (FC comes from React so import it)

**Review by amit-sawhney — APPROVED** [review]
looks fine, just typing could be improved

**Comment by eeshabarua** [comment]
Yes i approve

**Comment by abetco** [comment]
i don't

**Comment by amit-sawhney** [comment]
damn guess i'll unmerge

**Pull request merged**